diagnostic.md

```diff
@@ -33,7 +40,9 @@ y = 10;
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
 <!-- solution below -->
-
+x = 3 and y = 10
+The x variable changes to 3 because you are reassigning its value.  The variable
+y is initially the value of 3, but then changed to the value of 10
 
 ## Question 4
 
```

Comment:
Looks like you might have missed this one:

weather = "sunny"; // this evaluates to "sunny' as `weather`'s value
 weather === "sunny"; //this evaluates to `true`